scripts/build-ckeditor5.mjs

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+import { rm, copyFile } from 'fs/promises';
+import upath from 'upath';
+import chalk from 'chalk';
+import { build } from '@ckeditor/ckeditor5-dev-build-tools';
+
+function dist( path ) {
+	return upath.join( process.cwd(), 'dist', path );
```

Comment:
Similar to `scripts/banner.mjs`. Could we avoid `process.cwd()` and instead, calculate a path to the root directory manually?